legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/gitlab/api/GitLabReviewApi.java

```diff
@@ -70,6 +73,36 @@ public GitLabReviewApi(GitLabUserContext userContext)
     public List<Review> getReviews(String projectId, ReviewState state, Iterable<String> revisionIds, Instant since, Instant until, Integer limit)
     {
         LegendSDLCServerException.validateNonNull(projectId, "projectId may not be null");
+        GitLabProjectId gitLabProjectId = parseProjectId(projectId);
+        return getReviews(gitLabProjectId, projectId,  state, revisionIds, since, until, limit, gitLabProjectId.getGitLabMode(), false);
+    }
+
+    @Override
+    public List<Review> getReviews(Boolean assignedToMe, ReviewState state, Instant since, Instant until, Integer limit, ProjectType projectType)
+    {
+        List<Review> reviews = new ArrayList<>();
+        List<GitLabMode> modes = new ArrayList<>();
+        assignedToMe = (assignedToMe == null) || assignedToMe;
+
+        if (projectType == null)
+        {
+            getValidGitLabModes().forEach(modes::add);
+        }
+        else
+        {
+            modes.add(getGitLabModeFromProjectType(projectType));
+        }
+
+        for (GitLabMode mode: modes)
+        {
+            reviews.addAll(getReviews(null, null, state, null, since, until, limit, mode, assignedToMe));
+        }
+
+        return reviews;
+    }
+
+    public List<Review> getReviews(GitLabProjectId gitLabProjectId, String projectId, ReviewState state, Iterable<String> revisionIds, Instant since, Instant until, Integer limit, GitLabMode gitLabMode, Boolean assignedToMe)
```

Comment:
This should be private. Also, since this is explicitly taking GitLabMode, it should not also take GitLabProjectId (which contains a GitLabMode). Instead, it should take the project id as an Integer. Also, those two parameters should be next to each other in the signature.